controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -161,6 +241,9 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(request ctrl.Reques
 
 	enactmentConditions := enactmentconditions.New(r.Client, nmstateapi.EnactmentKey(nodeName, instance.Name))
 
+	if !instance.Spec.Parallel {
+		defer r.releaseNodeRunningUpdate(request.NamespacedName)
```

Comment:
We have to move this to the other Parallel if statement block

Reply:
Done